src/components/model/model.ts

```diff
@@ -0,0 +1,180 @@
+import View from '../view/view';
+import Controller from '../controller/controller';
+import ICar from '../controller/ICar';
+import IWinner from '../controller/IWinner';
+
+class Model {
+    view: View;
+    controller: Controller;
+    constructor() {
+        this.view = new View();
+        this.controller = new Controller();
+    }
+
+    start() {
+        this.controller.getWinners((data: IWinner[]) => this.view.winners.fillTable(data));
+        this.controller.getCars((data: ICar[]) =>
+            this.view.track.drawCars(data, (car: HTMLDivElement) => this.assignCarListeners(car))
+        );
+        this.addListeners();
+    }
+
+    addListeners() {
+        const garageButton = <HTMLButtonElement>document.querySelector('.garage');
```

Comment:
You should avoid using queryselectors, all controls that you need access to can be stored in view as properties upon creation